File: java2csharp/java/Samsung/Chord/Adapters/Item.java

package com.samsung.android.sdk.chord.example.adapter;

import android.graphics.Bitmap;

public class Item {

    Bitmap image;

    String text;

    public Item(Bitmap img, String txt) {
        this.image = img;
        this.text = txt;
    }

    public Bitmap getImage() {
        return image;
    }

    public void setImage(Bitmap image) {
        this.image = image;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}

File: java2csharp/csharp/Samsung/Chord/Item.cs
namespace com.samsung.android.sdk.chord.example.adapter
{

	using Bitmap = android.graphics.Bitmap;

	public class Item
	{

		internal Bitmap image;

		internal string text;

		public Item(Bitmap img, string txt)
		{
			this.image = img;
			this.text = txt;
		}

		public virtual Bitmap Image
		{
			get
			{
				return image;
			}
			set
			{
				this.image = value;
			}
		}


		public virtual string Text
		{
			get
			{
				return text;
			}
			set
			{
				this.text = value;
			}
		}


	}

}